azure-pipelines.yml

```diff
@@ -190,22 +191,16 @@ stages:
         LAPACK: None
         ATLAS: None
         CC: /usr/bin/clang
-      condition: eq(variables['USE_OPENBLAS'], '1')
-    - script: python setup.py build -j 4 build_ext --inplace install
-      displayName: 'Build NumPy without OpenBLAS and new casting'
-      env:
-        BLAS: None
-        LAPACK: None
-        ATLAS: None
-        CC: /usr/bin/clang
-      condition: eq(variables['USE_OPENBLAS'], '0')
```

Comment:
At one point these were different, but then we removed an env variable so they became the same